src/react/components/pages/predict/predictPage.tsx

```diff
@@ -377,6 +396,63 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
         return 1;
     }
 
+    private handleDownloadClick = () => {
+        this.triggerDownload();
+    }
+
+    private async triggerDownload(): Promise<any> {
+        axios.get("/analyze.py").then((response) => {
```

Comment:
Could place script in json as a string instead of getting it from the public folder. I'm not sure which is more ideal